security/pkg/nodeagent/test/success_sds/doc.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package successsds
```

Comment:
why do we need this?

Reply:
This is to test the SDS behavior between Envoy and SDS server, and helps to catch issue and debug. It just catches a race condition issue in the SDS server code.

Reply:
Is this just a stub file not implemented?